src/gengowatcher/ui_textual.py

```
@@ -655,6 +656,28 @@ class ConfigPreview(DashboardQuadrant):
+        "EmailMonitor",
+        "WebsiteMonitor",
+        "AutoAccept",
+        "HighValue",
+        "Cancellation",
+        "Network",
+        "Paths",
+        "Logging",
+        "DebugCategories",
+        "RateLimit",
+        "WebServer",
+    ]
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)